### ISSUE TITLE
starting to use dbdeployer by adding a remote download fails

### ISSUE THREAD

**Issue by lefred** [datacharmer/dbdeployer#137]
**Describe the bug**
Just after having installed dbdeployer, if the first command it to add a remote package to the download list, the operation will fail as ~/.dbdeployer is not existing.

**To Reproduce**
1. start from scratch (no ~/.dbdeployer directory)
2. download dbdeployer
3. run `dbdeployer downloads add-remote mysql 8.0 linux`

the output is:

```
[fred@fedora ~] $ dbdeployer downloads add-remote mysql 8.0 linux
error writing tarball list: error creating file /home/fred/.dbdeployer/tarball-list.json: open /home/fred/.dbdeployer/tarball-list.json: no such file or directory
```
if I create the directory manually (I didn't run `dbdeployer init`), it works has expected:

```
[fred@fedora ~] $ dbdeployer downloads add-remote mysql 8.0 linux
Tarball below added to /home/fred/.dbdeployer/tarball-list.json
Name:          mysql-8.0.26-linux-glibc2.12-x86_64.tar.xz
Short version: 8.0
Version:       8.0.26
Flavor:        mysql
OS:            Linux
URL:           https://dev.mysql.com/get/Downloads/MySQL-8.0/mysql-8.0.26-linux-glibc2.12-x86_64.tar.xz
Checksum:      MD5:100a0e9336ef106a5fe90e6803b57066
Size:          915 MB
Notes:         added with version 1.62.0
Added on:      2021-09-12 10:55
Name:          mysql-8.0.26-linux-glibc2.17-x86_64-minimal-rebuild.tar.xz
Short version: 8.0
Version:       8.0.26
Flavor:        mysql
OS:            Linux
URL:           https://dev.mysql.com/get/Downloads/MySQL-8.0/mysql-8.0.26-linux-glibc2.17-x86_64-minimal-rebuild.tar.xz
Checksum:      MD5:d830538cb6644100275a013541a9f129
Size:          52 MB
Notes:         added with version 1.62.0
Added on:      2021-09-12 10:55
```

**Expected behavior**
If the folder is not present, dbdeployer should create it or ask to run init if init creates it.

**Environment:**
 - OS: Linux Fedore 34
 - dbdeployer version : 1.62.0

**Comment by datacharmer** [comment]
Thanks for this useful finding. I will fix it soon

**Comment by datacharmer** [comment]
The reason for the bug is that dbdeployer incorrectly assumed that `$HOME/.dbdeployer` exists, but that is true only if at least one sandbox has been deployed.
The fix requires a check before downloads operations.

**Comment by datacharmer** [comment]
Fixed in [v1.63.0](https://github.com/datacharmer/dbdeployer/releases/tag/v1.63.0)
